library/src/tensile_host.cpp

```diff
@@ -211,6 +213,31 @@ static auto GetTensileInputs(const RocblasContractionProblem<T, U, V>& problem)
     return inputs;
 }
 
+bool TestPath(std::string path)
+{
+  struct stat st;
+  if (stat(path.c_str(), &st) == 0)
```

Comment:
`access()` is probably better than `stat()`. Also, should use `const std::string& path`, like:

```
inline bool TestPath(const std::string& path)
{
    return access(path.c_str(), R_OK) == 0;
}
```